src/app/components/ContractVerificationIcon/index.tsx

```diff
@@ -84,14 +86,21 @@ export const VerificationIcon: FC<{ address_eth: string; verified: boolean; noLi
     verified: t('contract.verification.isVerified'),
     unverified: t('contract.verification.isNotVerified'),
   }
+  const linkProps = {
+    href: `https://sourcify.dev/#/lookup/${address_eth}`,
+    rel: 'noopener noreferrer',
+    target: '_blank',
+    sx: { fontWeight: 400, color: 'inherit', textDecoration: 'underline' },
```

Comment:
It looks different on tokens list:
https://ee6efc6e.oasis-explorer.pages.dev/mainnet/emerald/token?page=2
![image](https://github.com/oasisprotocol/explorer/assets/3758846/49f17329-a31c-4d6c-a54f-5343c5ea1ec4)

Reply:
need to validate with Don when we want a link and when a badge.